modules/subsystems/module.go

```diff
@@ -25,10 +26,22 @@ var (
 )
 
 func init() {
-	// enable partial starting
+	// The subsystem layer takes over module management. Note that
+	// no one must have called EnableModuleManagement. Otherwise
+	// the subsystem layer will silently fail managing module
+	// dependencies!
+	// TODO(ppacher): we SHOULD panic here!
+	// TASK(#1431)
 	modules.EnableModuleManagement(handleModuleChanges)
 
 	// register module and enable it for starting
+	// TODO(ppacher): we have a very problematic dependecy to base here:
+	// 				  the "base" module is actually part of portmaster
+	//				  which in turn depends on portbase. This works as there's
+	//				  no import cycle but it basically makes the subsystems package
+	//                usable without bringing in a dependency on the complete portmaster!
+	//			      Consider getting rid of the "base" dependency or moving the whole
+	//				  subsystem layer to portmaster.
```

Comment:
This is actually intended, as everyone who starts using portbase should bring their own base module. The base module is primarily used for setup procedures that have to happen before anything else. If that is not needed, a developer can supply a no-op base module. I admit, this is not documented anywhere, afaik.